src/policy_provider/SamplePolicyProvider.c

```diff
@@ -271,13 +318,17 @@ int BSLP_FinalizePolicy(const void *user_data, const BSL_SecurityActionSet_t *ou
                     break;
                 }
             }
+
+            if (conclusion != BSL_SECOP_CONCLUSION_SUCCESS)
+            {
+                error_ret = BSLP_PolicyProvider_HandleFailures((BSL_BundleRef_t *)bundle, secop);
+            }
         }
     }
     (void)user_data;
     (void)output_action_set;
     (void)response_output;
```

Comment:
Replace these with the `arg _U_` macro for unused arguments in the function definition signature.